api/PclusterApiHandler.py

```diff
@@ -162,13 +147,29 @@ def _wrapper_authenticated(*args, **kwargs):
 
     return _authenticated
 
+def get_scopes_list():
+  if not SCOPES_LIST:
+    return "openid"
+  elif "openid" not in SCOPES_LIST:
+    return SCOPES_LIST + " openid"
+  return SCOPES_LIST
 
+def get_redirect_url():
+  return f"{SITE_URL}/login"
+  
 # Local Endpoints
 
 
 def get_version():
     return {"version": API_VERSION, "enable_mfa": ENABLE_MFA == "true"}
 
+def get_app_config():
+  return {
+    "auth_path": AUTH_PATH,
```

Comment:
issue(blocking): please use "AUTH_URL" instead of "AUTH_PATH", since the URL to redirect the user to, in order to perform the login" is in that env var

Reply:
Just to we are clear, `f'{AUTH_URL}'` is `f'{AUTH_PATH}/login'`, you are saying that the url should contain the `login` bit @BarcoMasile

Reply:
I'm saying the opposite
"/login" should not be added manually by the frontend (see my other [comment](https://github.com/aws-samples/pcluster-manager/pull/220#discussion_r930046830)) since it is not mandatory that OIDC provider auth urls end in "/login"

Reply:
But then `$AUTH_PATH` should be correct here, no?

Reply:
Synced up offline. The point here was that `/login` is used only bi Cognito (our default), but not by other providers, and we should use the value provided by the customer in the ENV varibale as is.